### ISSUE TITLE
mute audio on live recording reloading the video

### ISSUE THREAD

**Issue by keval101** [collab-project/videojs-record#603]
## Description

I want  mute functionality on live video but when I change audio: false in config, live recording is started again 

## Steps to reproduce
 this.config = {
      controls: false,
      autoplay: false,
      fluid: false,
      loop: false,
      width: 320,
      height: 240,
      bigPlayButton: false,
      controlBar: {
        fullscreenToggle: false,
        volumePanel: false,
        recordIndicator: false,
      },
      plugins:{
        // configure videojs-record plugin
        record: {
          audio: true,
          video: {
            facingMode: 'environment',
          },
          debug: true,
          maxLength: 10000
        }
      }
    };
  }

this.headerService.muteUnmuteMic.subscribe(
      res =>{
        console.log(res)
        this.micValue = res
        console.log(this.micValue)
        this.config.plugins.record.audio = this.micValue
       
      })

## Results

When I change audio in config, it reload the recording.

### Expected
It should not reload recording. It should disable/enable audio while recording is going on

### Actual
Please describe what actually happened.

### Error output
If there are any errors at all, please include them here.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
what version of videojs does this occur with?
#### browsers
what browser(s) are affected? Make sure to test with all third-party browser **extensions disabled**.
#### OSes
what platforms (operating systems and devices) are affected?

**Comment by thijstriemstra** [comment]
> I want  mute functionality on live video 

Can you clarify? Sounds like you want a video-only recording?

**Comment by keval101** [comment]
No... I want mute and unmute audio functionality while video recording.

**Comment by z639** [comment]
Keep the constraints to true. ie: audio : true and mute the track.

`player.record().stream.getTracks().forEach((track,) => {
        if (track.kind === "audio") {
            track.enabled = false; // false = muted, true = not muted.
        }
    }); `
